.bcr/config.yml

```diff
@@ -0,0 +1,4 @@
+# See https://github.com/bazel-contrib/publish-to-bcr#a-note-on-release-automation
+fixedReleaser:
+  login: mollyibot
+  email: mollyibot@google.com
```

Comment:
you need to set `moduleRoots` here: https://github.com/bazel-contrib/publish-to-bcr/tree/main/templates#metadatatemplatejson